hovercraft/generate.py.orig

```diff
@@ -0,0 +1,234 @@
+import os
```

Comment:
Get rid of all orig and BACKUP files as well.

In general you should look through the pull request diff before you make a pull request so you know what you are pull requesting.